src/app/organizations/settings/organization-subscription.component.ts

```diff
@@ -156,6 +157,21 @@ export class OrganizationSubscriptionComponent implements OnInit {
         }
     }
 
+    async removeSponsorship() {
+        const isConfirmed = await this.platformUtilsService.showDialog(
+            this.i18nService.t('removeSponsorshipConfirmation'),
+            this.i18nService.t('removeSponsorship'),
+            this.i18nService.t('remove'), this.i18nService.t('cancel'), 'warning');
+
+        if (!isConfirmed) {
+            return;
+        }
+        this.removeSponsorshipPromise = this.apiService.deleteRemoveSponsorship(this.organizationId);
+        await this.removeSponsorshipPromise;
```

Comment:
Should we have an error handler? To show an error toast

Reply:
I just put it into a `try catch` but I believe the promise in conjunction with the `appApiAction` will show a toast on the error.